vispy/visuals/image.py

```diff
@@ -87,6 +87,21 @@
         return texture2D($texture, texcoord);
     }"""
 
+import array, struct
+
+def floatToFixed(f):
+    str = array.array('f', f.ravel()).tostring()
+    print(len(str))
+    size = f.size * 4
+    fmt = "!%dB" % size
+    print(fmt, size, f.shape + (4,))
```

Comment:
There is probably a one-liner way to do this with `numpy` directly, like `np.fromstring(np.array(f, np.float32).tostring(), 'b')` or so